FastAPI Assignment 2/app/schemas/schema.py

```diff
@@ -0,0 +1,65 @@
+from pydantic import BaseModel, Field, EmailStr, validator
+from enum import Enum
+
+
+class SongDetailSchema(BaseModel):
+    artist_id: int
+    album: str = Field(default=None)
+    title: str = Field(default=None)
+    genre: str = Field(default=None)
+
+    class Config:
+        schema_extra = {
+            "song_demo": {
+                "artist_id": 1,
+                "album": "Purpose",
+                "title": "What do you mean",
+                "genre": "Acoustic Blues",
+            }
+        }
+
+
+class SongUpdateSchema(BaseModel):
+    title: str = Field(default=None)
+    album: str = Field(default=None)
+    genre: str = Field(default=None)
+
+
+class Roles(str, Enum):
+    user = "user"
+    admin = "admin"
+
+
+class UserSchema(BaseModel):
+    fullname: str = Field(default=None)
+    email: EmailStr = Field(default=None)
+    password: str = Field(default=None, min_length=5)
```

Comment:
Pydantic provides a type for accepting secret values called `SecretStr`. use this to avoid leaking secrets.